### ISSUE TITLE
Player can start a game without inputing required info

### ISSUE THREAD

**Issue by mdctenga** [devleague/jog-it-off#28]
Objective:
Host player cannot start game until the following fields from /create are filled:
- Room Name
- Number of Points per Player
- Plot Time (minutes)
- Game Time (minutes)

Steps to Reproduce Bug:
1) Navigate to "Create A Game!" view
2) Input anything in "Room Name" field
3) Press "Enter" key on your keyboard instead of inputting anything in any other field
- Notice how the game still gets created

What should happen:
- Validate that each field has acceptable input for all 4 input fields
- If any input field is empty, display error message upon "Submit" button click
  -- "Room Name" can take any input.
  -- "# of Points per Player" can only take numbers. If input is not number, display error "must be a number"
  -- "Plot Time (minutes)" can only take numbers. If input is not number, display error "must be a number"
  -- "Game Time (minutes)" can only take numbers. If input is not number, display error "must be a number"

**Comment by cannonbrooke** [comment]
Got it to work on the client side, tony suggested server side but tony isn't writing the code and it can wait. You have to enter text for it to work; however, if we want fancy text we need to look into foundation. That can wait. Maybe stretch goal. 
- Cannon
